CHANGES.md

```diff
@@ -1,5 +1,6 @@
 ### Apache MetaModel _WIP_
 
+ * [METAMODEL-1219] - Improved returning schemas for SQL Server
```

Comment:
The pull request looks good to me, it can't be merged however, because this change conflicts with another update to the CHANGES.md file on apache:master. Can you merge apache:master into your branch?

Reply:
Resolved the conflict